data/com.github.gnotime.gnotime.gschema.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schemalist gettext-domain="gnotime-2.0">
+  <schema id="com.github.gnotime.gnotime" path="/com/github/gnotime/gnotime/">
+    <child name="geometry" schema="com.github.gnotime.gnotime.geometry"/>
+    <child name="toolbar" schema="com.github.gnotime.gnotime.toolbar"/>
+  </schema>
+
+  <schema id="com.github.gnotime.gnotime.geometry" path="/com/github/gnotime/gnotime/geometry/">
```

Comment:
Is it customary to involve "github" like that, just because we happen to host the source at github?

Either way, I went ahead and secured the domain name gnotime.org for the project in case we want it. So that's an option. For now, it's just a redirect to https://gttr.sourceforge.net/ (might not be fully active at time of writing).

Reply:
Wow, that's great, that you secured the domain name! I just followed the suggestions of Gnome for coming up with an application id ([link](https://developer.gnome.org/documentation/tutorials/application-id.html)). Now that you secured the domain I went with `org.gnotime.app`. By the additional `app` level I want to leave space for future extensions (e.g. a plug-in system). Would you be fine with this application id (since this is a quite fundamental choice)?

Reply:
Yes, that sounds good. Their main naming scheme is basically "organisation name" followed by "application name" and I *don't* think we should use only "org.gnotime" just because the app and the organisation happens to have the same name. So `org.gnotime.app` looks good to me.